src/content/docs/synthetics/synthetic-monitoring/private-locations/verified-script-execution-private-locations.mdx

```diff
@@ -10,7 +10,7 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-To prevent others from using your private minions to assign [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-scripted-browsers) or [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-api-tests), add verified script execution. Then, any changes to your minions will require a passphrase that is known only to you.
+To prevent others from using your private minions or synthetic job managers to assign [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-scripted-browsers) or [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-api-tests), add verified script execution. Then, any changes to your minions will require a passphrase that is known only to you.
```

Comment:
@sadafarshad I'm still not clear about what this means. Does this edit see accurate:

```
To prevent others from using your private minions or to prevent synthetic job managers from assigning [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-scripted-browsers) or [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-api-tests), add verified script execution.
```

Reply:
Would this sound better: 

`To prevent others from using your private minions and synthetic job managers for assigning [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-scripted-browsers) or [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-api-tests), add verified script execution. Then, any changes to your minions will require a passphrase that is known only to you.`

Reply:
Essentially, we want users to use verified script execution if they want to prevent others from using their private minions or synthetic job managers to assign scripted browser and API tests.

Reply:
@sadafarshad let's keep your original sentence! Sorry for the confusion!